packages/aws-cdk/lib/init-templates/sample-app/javascript/.template.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+
+# CDK asset staging directory
+.cdk.staging
```

Comment:
Do you mind handling #2536 while you are at it?

Reply:
No problem

Reply:
@eladb, sorry I don't actually understand your requirement, do you want `.cdk.staging` to be removed from  .template.gitignore and .template.npmignore?

Reply:
No, added to the typescript project templates like you did here.

Reply:
They are there already: 
- https://github.com/awslabs/aws-cdk/blob/master/packages/aws-cdk/lib/init-templates/sample-app/typescript/.template.gitignore#L6
- https://github.com/awslabs/aws-cdk/blob/master/packages/aws-cdk/lib/init-templates/sample-app/typescript/.template.npmignore#L5

Reply:
Ha. Sorry!